platform/security/certificate/security-certificate-keystoreeditor/src/test/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditorTest.java

```diff
@@ -179,6 +193,81 @@ public void setup() throws IOException {
     }
 
     @Test
+    public void testShowCertificateFromUrl()
+            throws IOException, KeystoreEditor.KeystoreEditorException {
+        KeystoreEditor keystoreEditor = getNonVerifyingKeystoreEditor();
+        addCannedCertificatesToTruststore(keystoreEditor);
+        byte[] encoded = Base64.getEncoder()
+                .encode("https://notarealurlatall.com".getBytes());
```

Comment:
I wonder if we should change URLs like this to https://doesnotexist.codice.org.  At least then we can make sure it will never exist.